pkgs/applications/audio/csound/default.nix

```diff
@@ -30,20 +25,18 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-O7s92N54+zIl07eIdK/puoSve/qJ3O01fTh0TP+VdZA=";
   };
 
-  cmakeFlags = [ "-DBUILD_CSOUND_AC=0" ] # fails to find Score.hpp
-    ++ lib.optional stdenv.isDarwin "-DCS_FRAMEWORK_DEST=${placeholder "out"}/lib"
+  cmakeFlags = lib.optional stdenv.isDarwin "-DCS_FRAMEWORK_DEST=${placeholder "out"}/lib"
     # Ignore gettext in CMAKE_PREFIX_PATH on cross to prevent find_program picking up the wrong gettext
     ++ lib.optional (stdenv.hostPlatform != stdenv.buildPlatform) "-DCMAKE_IGNORE_PATH=${lib.getBin gettext}/bin";
```

Comment:
Can you use `canExecute` instead?

Reply:
OK, will have a look, thanks.

Reply:
`!(stdenv.buildPlatform.canExecute stdenv.hostPlatform)` has a slightly different meaning to `stdenv.hostPlatform != stdenv.buildPlatform`. I don't want to change this and break something.

@lilyinstarlight added it in #227135.

I have tried building `.#legacyPackages.x86_64-linux.pkgsCross.aarch64-multiplatform.{csound,csound-plugins}` to test it. Both can build with that cmake flag completely removed.